AcsEmulator/AcsEmulatorAPI/CallAutomationController.cs

```diff
@@ -9,21 +9,53 @@ public static class CallAutomationController
     {
         public static void AddCallAutomationEndpoints(this WebApplication app)
         {
-            app.MapPost("/calling/callConnections", [Authorize] async (ClaimsPrincipal principal, AcsDbContext db, CreateCallRequest req) =>
+            app.MapPost("/calling/callConnections", async (AcsDbContext db, CreateCallRequest req) =>
             {
                 // todo: generate proper id, save to db
                 var callConnectionId = Guid.NewGuid().ToString();
-                var result = new
+                var result = new CallConnectionProperties
                 {
-                    CallConnectionProperties = new
-                    {
-                        AnsweredBy = Guid.NewGuid(),
-                        CallConnectionId = callConnectionId,
-                        req.CallbackUri
-                    }
+                    CallConnectionId = callConnectionId,
+                    CallbackUri = req.CallbackUri,
+                    CallConnectionState = CallConnectionStateModel.Connected,
+                    ServerCallId = "serverCallId456",
+                    Source = new CommunicationIdentifierModel("communicationUser", new CommunicationUser("source_id")),
+                    Targets = req.Targets    
                 };
+
                 return Results.Created($"/calling/callConnections/{callConnectionId}", result);
             });
+
+            app.MapGet("/calling/callConnections/{callConnectionId}", async (AcsDbContext db, string callConnectionId) =>
+            {
+                // // Retrieve call connection information from the database based on the provided callConnectionId
+                // var callConnection = await db.CallConnections.FindAsync(callConnectionId);
+    
+
+                // if (callConnection == null)
+                // {
+                //     return Results.NotFound("Call connection not found.");
+                // }
+
+                // Prepare the response containing the CallConnectionProperties
+                var callConnectionProperties = new CallConnectionProperties
+                    {
+                        AnsweredBy = new CommunicationUser("answer_id"),
+                        CallConnectionId = callConnectionId,
+                        CallbackUri = "https://example.com/callback",
+                        CallConnectionState = CallConnectionStateModel.Connected,
+                        CorrelationId = "correlationId123",
+                        ServerCallId = "serverCallId456",
+                        Source = new CommunicationIdentifierModel("communicationUser", new CommunicationUser("source_id")),
```

Comment:
Most of these response properties should get populated from what was in the request